### PR TITLE
Narrow logits to featuremaps dimensions

### PR DESCRIPTION
**Pull request by karakusc**
Fixes convergence on bs=2. 

[0418 07:49:23 @monitor.py:467] mAP(bbox)/IoU=0.5: 0.59152
[0418 07:49:23 @monitor.py:467] mAP(bbox)/IoU=0.5:0.95: 0.37334
[0418 07:49:23 @monitor.py:467] mAP(bbox)/IoU=0.75: 0.40401
[0418 07:49:23 @monitor.py:467] mAP(bbox)/large: 0.4812
[0418 07:49:23 @monitor.py:467] mAP(bbox)/medium: 0.40785
[0418 07:49:23 @monitor.py:467] mAP(bbox)/small: 0.21708
[0418 07:49:23 @monitor.py:467] mAP(segm)/IoU=0.5: 0.55663
[0418 07:49:23 @monitor.py:467] mAP(segm)/IoU=0.5:0.95: 0.34286
[0418 07:49:23 @monitor.py:467] mAP(segm)/IoU=0.75: 0.36358
[0418 07:49:23 @monitor.py:467] mAP(segm)/large: 0.46205
[0418 07:49:23 @monitor.py:467] mAP(segm)/medium: 0.37592
[0418 07:49:23 @monitor.py:467] mAP(segm)/small: 0.18124

### CODE REVIEW

**Comment by armandmcqueen** [comment]
Looks good. Approved
